### PR TITLE
ENG-3267 | Removed Print statement and returning response

### PR DESCRIPTION
**Pull request by ankitintg**
This PR is raised against removing the print statement and return the original response when getting any error-
Reference ticket- https://sightmachine.atlassian.net/browse/ENG-3267

### CODE REVIEW

File: smsdk/ma_session.py [hunk 1]
```diff
@@ -248,15 +248,9 @@ def _complete_async_task(
                     if state == "FAILURE" or state == "REVOKED":
                         raise ValueError("Error - {}".format(response.text))
                 except:
```
Comment (mgoldmansight) [inline smsdk/ma_session.py:250]:
```suggestion
                except Exception as e:
                  raise e
```
This doesn't feel like returning the response is fixing the issue the ticket suggests that we should be raising the actually error.  Maybe print the error as log line as well.

File: smsdk/ma_session.py [hunk 2]
```diff
@@ -248,15 +248,9 @@ def _complete_async_task(
...
-
-                    traceback.print_exc()
-                    return []
+                    return response.text
         except:
-            import traceback
-
-            traceback.print_exc()
-            return []
+            return response.text
```
Comment (mgoldmansight) [inline smsdk/ma_session.py:253]: Same as above

File: smsdk/ma_session.py (hunk 1 above)
Comment (ankitintg) [reply smsdk/ma_session.py]: Done

File: smsdk/ma_session.py (hunk 2 above)
Comment (ankitintg) [reply smsdk/ma_session.py]: I did the changes as required.

**Comment by ankitintg** [comment]
@mgoldmansight, Requesting you to please have a look at the review comments I've resolved.

**Review by mgoldmansight — APPROVED** [review]

**Pull request merged**